### PR TITLE
add Youtube and Gmail for account deletion data

### PR DESCRIPTION
**Pull request by Zyd8**
-- can't see the newly icon being reflected on local dev frontend since it refers to what is in the online github repository.

### CODE REVIEW

**Comment by Zyd8** [comment]
Hello there, Julian. Hope, you're having a nice day. Do let me know if you have any issues with the changes I've done.

File: data/services.json [hunk 1]
```diff
@@ -98,5 +98,15 @@
     "name": "Strava",
     "icon": "strava",
     "deleteAccountPage": "https://www.strava.com/athlete/delete_your_account"
+  },
+  {
+    "name": "Youtube",
```
Comment (j4n-e4t) [inline data/services.json:103]:
```suggestion
    "name": "YouTube",
```

**Review by j4n-e4t — CHANGES_REQUESTED** [review]
Hi @Zyd8, thank you so much for contributing! I honestly thought no one would actually care about projects like this one but it's amazing to be proven wrong. This PR really made my day. Thank you! 🎉

I would change the spelling of YouTube to camel case since it's the official style. Other than that, awesome PR. 👍

**Comment by j4n-e4t** [comment]
> -- can't see the newly icon being reflected on local dev frontend since it refers to what is in the online github repository.

Also, I'm searching for a better way of hosting the images. GitHub isn't ideal for that exact reason. 😉

**Comment by Zyd8** [comment]
> Hi @Zyd8, thank you so much for contributing! I honestly thought no one would actually care about projects like this one but it's amazing to be proven wrong. This PR really made my day. Thank you! 🎉
> 
> I would change the spelling of YouTube to camel case since it's the official style. Other than that, awesome PR. 👍

@j4n-e4t Thank you also for making me contribute! I just love your project since I have _too much_ digital footprint in the internet. Stay tuned for more PRs!

**Pull request merged**